soc/arm/st_stm32/stm32h7/Kconfig.defconfig.stm32h723xx

```diff
@@ -0,0 +1,15 @@
+# ST STM32H723X MCU configuration options
+
+# Copyright (c) 2020 Alexander Kozhinov
+# AlexanderKozhinov@yandex.com
```

Comment:
Would you mind using one liner format ?
I initially though this was required because of 80 char limits, but it appears to fit, so let's keep the usual format

Reply:
Done